simulator_service/launch/start_simulation_server.launch

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" ?>
+
+<!-- 
+  This launch file brings up a series of nodes in a special namespace
+  that simulate the same robot loaded normally.
+ -->
+
+<launch>
+
+  <arg name="namespace" default="simulation"/>
+  <arg name="world_frame" default="world_frame"/>
+  <arg name="service_name" default="simulate_trajectory"/>
+  <arg name="action_name" default="follow_joint_trajectory"/>
+  <!-- The default scale factor is multiplied by trajectory durations to get the final timing -->
+  <arg name="scale_factor" default="0.2"/>
+  
+  <group ns="$(arg namespace)">
+
+    <!-- publish the robot state (tf transforms) in a namespaced topic -->
+    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
+      <param name="tf_prefix" value="$(arg namespace)"/>
+    </node>
+
+    <!-- Connects the simulation world_frame with the real world_frame so rviz can visualize both at once -->
+    <node pkg="tf" type="static_transform_publisher" name="world_to_sim_publisher" 
+      args="0 0 0 0 0 0 
+            $(arg world_frame) $(arg namespace)/$(arg world_frame) 1000"/>
```

Comment:
Quick note: the last parameter is the period, not the frequency/rate. Was this set intentionally at 1 Hz?